DDInstall/DDInstall.psm1

```diff
@@ -0,0 +1,192 @@
+function Install-DDAgent
+{
+    param (
+        [Parameter()][ValidateSet('DotNet')][String[]]$WithAPMTracers,
+        [String] $AgentInstallURL,
```

Comment:
We should support a file path, too. in case an admin stores the MSI on a file share, or is running this module as part of a bundle, like from SCCM, and the MSI file is included alongside it.

Reply:
good thinking!